### PR TITLE
OGD-491 adding payload field for self employment income

### PR DESCRIPTION
**Pull request by edgarjimenez**

### CODE REVIEW

File: resources/public/api/conf/1.0/schemas/create-self-assessment-response.json [hunk 1]
```diff
@@ -45,6 +45,12 @@
             "type": "number",
             "description": "The income obtained from employments",
             "example": "13567.77"
+          },
+          "selfEmploymentProfit": {
+            "id": "/properties/saReturns/items/selfEmploymentProfit",
+            "type": "number",
+            "description": "The income obtained for the self employment",
```
Comment (tom59) [inline resources/public/api/conf/1.0/schemas/create-self-assessment-response.json]: the profit

Comment (edgarjimenez) [reply resources/public/api/conf/1.0/schemas/create-self-assessment-response.json]: That's what I had at first, but it doesn't make sense to me. Profit means something completely different.

Comment (tom59) [reply resources/public/api/conf/1.0/schemas/create-self-assessment-response.json]: This is what we are returning though. We may want to add later on the income and expenses.

Comment (edgarjimenez) [reply resources/public/api/conf/1.0/schemas/create-self-assessment-response.json]: done

**Pull request merged**